text-ui-test/EXPECTED.TXT

```diff
@@ -31,16 +31,25 @@ Add a new transaction entry, which could be either an "expense" or "income" into
 Usage: add t/TYPE c/CATEGORY a/AMOUNT d/DATE i/DESCRIPTION
 
 Command Word: LIST
-List all or some transactions based on selection. If tag filters are used, the transactions retrieved from the records must match all the filter tags that have been specified in order to be recognized as a valid record. The m/MONTH and y/YEAR tags should not be used together with p/PERIOD and n/NUMBER tags.
-Usage: list [t/TYPE] [c/CATEGORY] [d/DATE] [m/MONTH] [y/YEAR] [p/PERIOD] [n/NUMBER]
+List all or some transactions based on selection.
+If tag filters are used, the transactions retrieved from the records must match all the filter tags
```

Comment:
I like this update that has made it seems much clearer. I will update the UG to reflect this example too.